rclpy/rclpy/validate_parameter_name.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rclpy.exceptions import InvalidParameterException
+from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
+
+
+def validate_parameter_name(name):
```

Comment:
```suggestion
def validate_parameter_name(name: str) -> bool:
```

Reply:
Addressed in https://github.com/ros2/rclpy/pull/325/commits/0925fe4e4a72a4b32ff1ec58790d73c4ba35838a.